e2e/vm/additional_disk_test.go

```diff
@@ -55,6 +56,15 @@ var testAdditionalDisk = func(o *option.Option) {
 			command.Run(o, "start", containerName)
 			gomega.Expect(command.StdoutStr(o, "exec", containerName, "cat", "/tmp/test.txt")).
 				Should(gomega.Equal("foo"))
+
+		})
+
+		ginkgo.It("finch vm init by the disk size for finch area configured in finchDiskSize of ~/.finch.yaml", func() {
+			resetVM(o, installed)
+			resetDisks(o, installed)
```

Comment:
will users lose their persistent data if they have to resize their disk?

Reply:
What is the resolution for this? Can this be run without `resetDisks` with a test that checks the containers prior to resize are available after resize. As mentioned in the [ticket](https://github.com/runfinch/finch/issues/275) ".After I update the value, stop the VM, and then start the VM, the value should take effect."